docs/installing-viper.rst

```diff
@@ -99,3 +115,12 @@ install and test commands
     ::
         python setup.py install
         python setup.py test
+******
+Docker
+******
+A Docker image which executes most of the installation steps is provided.
+To ensure that everything works correctly after the installtion, please run the test commands
+and try compiling a contract:
+::
```

Comment:
It'd be great if you could explain how to setup docker starting with cloning the repo (e.i. `git clone ...`).